bls-runtime/src/cli_clap.rs

```diff
@@ -37,6 +41,12 @@ const PERMISSION_HELP: &str = "the permissions for app";
 
 const MODULES_HELP: &str = "the modules used by app";
 
+const STDOUT_HELP: &str = "the stdout setting by app, the value is inherit/null/the name of file";
+
+const STDERR_HELP: &str = "the stderr setting by app, the value is inherit/null/the name of file";
+
+const STDIN_HELP: &str = "the stdin setting by app, is inherit/null/string for input";
```

Comment:
nit: could improve working here; hard to understand what this means

Reply:
yes. I think so.